src/layouts/DefaultLayout.vue

```diff
@@ -70,14 +70,16 @@
             >
               <v-list density="compact">
                 <v-list-item
-                  v-for="item in (configStore.general as any).helpMenu.url"
+                  v-if="helpMenu"
+                  v-for="item in helpMenu.url"
                   :href="item.url"
                   target="_blank"
                   append-icon="mdi-open-in-new"
                   >{{ item.name }}</v-list-item
                 >
                 <v-list-item
-                  v-for="item in (configStore.general as any).helpMenu.path"
+                  v-if="helpMenu"
+                  v-for="item in helpMenu.path"
                   :to="{ name: 'HtmlDisplay', params: { path: item.path } }"
                   href="#"
                   >{{ item.name }}</v-list-item
```

Comment:
Just a note (seperate from this issue), but this currently always orders the urls first followed by the paths.
This would have to be changed in the backend though:
![image](https://github.com/Deltares/fews-web-oc/assets/144685504/7ebb8b01-4a1e-4bc4-ad85-1fce6a810d43)
Where the items should be able to be ordered interleaved with a type param specifying the type.

Reply:
that would be nice but you would still need to have 2 `v-list-item` because you can't specify both `href` and `to` attributes.

Reply:
Yes, the v-for would then probably be on a `<template>` parent